basculehttp/mocks_test.go

```diff
@@ -49,3 +54,52 @@ func (p *mockParser) ParseJWT(token string, claims jwt.Claims, parseFunc jwt.Key
 	_, err = parseFunc(t)
 	return t, err
 }
+
+// MockKey is a mock for key.
+type MockKey struct {
```

Comment:
There's no harm in having this be an exported type from a `_test` file.  But, in general, it's good to leave mocks as unexported, e.g. `mockKey`.  That makes it clearer to new developers that you never intend this type to be used in tests outside this package.

Reply:
Got it. Updated in my recent commit.